content/guides/development.md

```diff
@@ -240,7 +240,149 @@ T> Now that your server is working, you might want to give [Hot Module Replaceme
 
 ### Using webpack-dev-middleware
 
-?> Familiar with `webpack-dev-middleware`? We need your help! Please submit a PR to fill in the missing instructions and example here. Make sure to keep it simple as this guide is intended for beginners.
+`webpack-dev-middleware` is a wrapper that will emit files processed by webpack to a server. This is used in `webpack-dev-server` internally, however it's available as a separate package to allow more custom setups if desired. We'll take a look at an example that combines webpack-dev-middleware with an express server.
+
+Let's install `express` and `webpack-dev-middleware` so we can get started:
+
+``` bash
+npm install --save-dev express webpack-dev-middleware
+```
+
+Now we need to make some adjustments to our webpack configuration file in order to make sure the middleware will function correctly:
+
+__webpack.config.js__
+
+``` diff
+  const path = require('path');
+  const HtmlWebpackPlugin = require('html-webpack-plugin');
+  const CleanWebpackPlugin = require('clean-webpack-plugin');
+
+  module.exports = {
+    entry: {
+      app: './src/index.js',
+      print: './src/print.js'
+    },
+    devtool: 'inline-source-map',
+    plugins: [
+      new CleanWebpackPlugin(['dist']),
+      new HtmlWebpackPlugin({
+        title: 'Output Management'
+      }),
+      new webpack.HotModuleReplacementPlugin()
+    ],
+    output: {
+      filename: '[name].bundle.js',
+      path: path.resolve(__dirname, 'dist'),
++     publicPath: 'http://localhost/'
```

Comment:
I would try just `publicPath: '/'` but if that didn't work then I guess this is fine. Frankly, I still find `publicPath` a bit confusing -- it almost seems like for fully qualified urls the first section, e.g. `http://localhost` is just ignored for generated bundles. I think maybe people use fully qualified URLs like `https://cdn.example.com/` so that if something not generated by webpack returns a 404, the dev-server will fall back to actual location.